ui/lib/config-ui/addon/components/login-settings/page/details.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Component from '@glimmer/component';
+import { service } from '@ember/service';
+
+/**
+ * @module Page::LoginSettingsRuleDetails
+ * Page::LoginSettingsRuleDetails components are used to display list of rules.
```

Comment:
Can you add more here? Such as where it's used and how? In [this](https://github.com/hashicorp/vault/blob/main/ui/lib/core/addon/components/shamir/form.js#L12) example the two additional lines add the context.